enhancements/ingress/customize-error-code-pages.md

```diff
@@ -0,0 +1,121 @@
+---
+title: customize-error-code-pages
+authors:
+  - "@miheer"
+reviewers:
+  - "@Miciah"
+  - "@danehans"
+  - "@knobunc"
+approvers:
+  - "@Miciah"
+  - "@danehans"
+  - "@knobunc"
+creation-date: 2021-03-07 
+last-updated: 2020-03-07
+status: implementable 
+see-also:
+replaces:
+superseded-by:
+---
+# Customize error code paged returned by the haproxy router
+## Release Signoff Checklist
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+## Summary
+There is no supported and reasonable method to customize the HAProxy 2.x error page as deployed in OpenShift.
+Customizing the HAProxy error page per our recommended procedure (documented here: https://access.redhat.com/solutions/3425711).
+does not work with OCP 4. It's also important that the right page as per user requirements is served at the right moment.
```

Comment:
> Customizing the HAProxy error page per our recommended procedure (documented here: https://access.redhat.com/solutions/3425711). does not work with OCP 4

This solution was never intended to work on OCP 4, right? Also, I would suggest removing the spurious period after the parenthesis.

Reply:
I will reword it. Yes it is not meant for OCP 4